GDJS/GDJS/Extensions/Builtin/CameraExtension.cpp

```diff
@@ -20,20 +20,39 @@ CameraExtension::CameraExtension() {
   GetAllActions()["CameraY"]  // Deprecated
       .SetFunctionName("gdjs.evtTools.camera.setCameraY")
       .SetGetter("gdjs.evtTools.camera.getCameraY");
-  GetAllActions()["SetCameraX"]
+  GetAllActions()["SetCameraX"]  // Deprecated
       .SetFunctionName("gdjs.evtTools.camera.setCameraX")
       .SetGetter("gdjs.evtTools.camera.getCameraX");
-  GetAllActions()["SetCameraY"]
+  GetAllActions()["SetCameraY"]  // Deprecated
       .SetFunctionName("gdjs.evtTools.camera.setCameraY")
       .SetGetter("gdjs.evtTools.camera.getCameraY");
+  GetAllActions()["CameraCenterX"]
+      .SetFunctionName("gdjs.evtTools.camera.setCameraX")
+      .SetGetter("gdjs.evtTools.camera.getCameraX");
+  GetAllActions()["CameraCenterY"]
```

Comment:
This is not obvious (I got bitten already by this), but this should be:

```suggestion
  GetAllActions()["SetCameraCenterX"]
      .SetFunctionName("gdjs.evtTools.camera.setCameraX")
      .SetGetter("gdjs.evtTools.camera.getCameraX");
  GetAllActions()["SetCameraCenterY"]
```

as the convention, enforced by AddExpressionAndConditiondAndAction is that an action starts with a verb: `Set`